src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSpark.java

```diff
@@ -34,12 +34,62 @@
 
 import java.util.*;
 
+/**
+ * <p>This tool is a Spark implementation of the tool MarkDuplicates in Picard allowing for better utilization
+ *    of available system resources to speed up duplicate marking.</p>
+ *
+ * <p>This tool locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are
+ *    defined as originating from a single fragment of DNA.  Duplicates can arise during sample preparation e.g. library
+ *    construction using PCR.  See also "<a href='https://broadinstitute.github.io/picard/command-line-overview.html#EstimateLibraryComplexity'>EstimateLibraryComplexity</a>" +
+ *    for additional notes on PCR duplication artifacts.  Duplicate reads can also result from a single amplification cluster,
+ *    incorrectly detected as multiple clusters by the optical sensor of the sequencing instrument.  These duplication artifacts are
+ *    referred to as optical duplicates.</p>
+ *
+ * <p>The MarkDuplicates tool works by comparing sequences in the 5 prime positions of both reads and read-pairs in a SAM/BAM file.
+ *    After duplicate reads arecollected, the tool differentiates the primary and duplicate reads using an algorithm that ranks
+ *    reads by the sums of their base-quality scores (default method).</p>
+ *
+ * <p>The tool's main output is a new SAM or BAM file, in which duplicates have been identified in the SAM flags field for each
+ *    read.  Duplicates are marked with the hexadecimal value of 0x0400, which corresponds to a decimal value of 1024.
+ *    If you are not familiar with this type of annotation, please see the following <a href='https://www.broadinstitute.org/gatk/blog?id=7019'>blog post</a> for additional information.</p>" +
+ *
+ * <p>Although the bitwise flag annotation indicates whether a read was marked as a duplicate, it does not identify the type of
+ *    duplicate.  To do this, a new tag called the duplicate type (DT) tag was recently added as an optional output in
+ *    the 'optional field' section of a SAM/BAM file.  Invoking the 'duplicate-tagging-policy' option,
```

Comment:
Have you checked that all of the Picard args mentioned here are actually present in the Spark version?

Reply:
yes, I have checked and updated the names involved.